src/polygon.c

```diff
@@ -1431,3 +1432,54 @@ spherepoly_add_points_finalize(PG_FUNCTION_ARGS)
 	}
 	PG_RETURN_POINTER(poly);
 }
+
+
+Datum
+spherepoly_is_convex(PG_FUNCTION_ARGS)
+{
+	Vector3D	u,
+				v,
+				vsu,
+				wsv,
+				crs;
+	float8		cur = 0,
+				prev = 0;
+	SPOLY		*poly = (SPOLY *) PG_GETARG_POINTER(0);
+
+	if (poly == NULL)
+	{
+		PG_RETURN_NULL();
```

Comment:
I would prefer to return false in this case because FALSE value makes sense for invalid or NULL arguments.